datasets/nasa-nex-gddp-cmip6/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM pccomponentstest.azurecr.io/pctasks-task-base
```

Comment:
It's a bit annoying, but for now we're copy-pasting the full dockerfile definition. These lines, before the `COPY` you have below: https://github.com/microsoft/planetary-computer-tasks/blob/ee9e462ace0b5a3505161e34067c24bb18369b86/datasets/sentinel-5p/Dockerfile#L1-L67

Ran into issues in the past with Python dependencies getting messed up.

Reply:
Updated.